tests/acceptance/WPFirstCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+class WPFirstCest
```

Comment:
Ok, I'll work on that. The  WP is not necessary. It was just a demo from the config. The name can be anything.  The user can generate the file with whatever name they want.